core/arch/arm/plat-stm32mp1/main.c

```diff
@@ -211,24 +211,22 @@ void main_secondary_init_gic(void)
 	stm32mp_register_online_cpu();
 }
 
-#ifndef CFG_EMBED_DTB
 static TEE_Result init_stm32mp1_drivers(void)
 {
+#ifndef CFG_EMBED_DTB
 	/* Without secure DTB support, some drivers must be inited */
 	stm32_etzpc_init(ETZPC_BASE);
+#endif
```

Comment:
Minor: `if (!IS_ENABLED(CFG_EMBED_DTB))`?

Reply:
right, I should get used to this way. Will fix (and wll try to remember for the next times).